src/robot/output/output.py

```diff
@@ -45,6 +46,9 @@ def close(self, result):
         self._xmllogger.close()
         LOGGER.unregister_logger(self._xmllogger)
         LOGGER.output_file('Output', self._settings['Output'])
+        for logger in self._registered_loggers:
+            logger.close()
+        LOGGER.unregister_logger(*self._registered_loggers)
```

Comment:
It would perhaps be a little more clear to format the above like this:

```python
for logger in self._registered_loggers:
    logger.close()
    LOGGER.unregister_logger(logger)
```